src/styles/FAQSection.css

```diff
@@ -79,3 +79,23 @@
 .itemBtnExpanded .chevron {
 	transform: rotate(180deg);
 }
+
+@media (max-width: 450px) {
+	.faq-header h2 {
+		font-size: 9vw;
+	}
+
+	.faq-header h3 {
+		font-size: 6vw;
+	}
+
+	.itemBtn {
+		padding: 3vw;
+		font-size: 5vw;
+	}
+
+	.itemPanel {
+		padding: 3vw 4vw;
+		font-size: 4vw;
+	}
```

Comment:
Please use `rem` as a unit here, just for consistency. I know throughout the codebase there's some differing things but I think we should try to keep with rem if possible. (the media query can remain define in px)

Reply:
Since we're switching to rem instead, if you're having trouble replicating what you currently have, you can also just have the text wrap on the image (while remaining at a larger font size) when the screen shrinks.

You're also free to be creative and move the event information to a different section if the screen becomes a certain size, so long as it makes sense to do so.